config/locales/en/claims/support/support_users.yml

```diff
@@ -3,7 +3,7 @@ en:
     support:
       support_users:
         index:
-          page_title: Users
+          page_title: Support users
```

Comment:
This is also an important piece of presentational content, we should have page title tests :)

Reply:
Can we use the same format as in `spec/system/claims/support_users/add_a_support_user_spec.rb`?

```ruby
and_the_page_title_is("Error: Personal details - Add user - Claim funding for mentor training")
```